README.md

```diff
@@ -40,20 +64,17 @@ aws ecs run-task --cluster wifi-api-cluster --task-definition logging-api-task-w
 aws ecs run-task --cluster wifi-api-cluster --task-definition logging-api-task-wifi --count 1 --overrides "{ \"containerOverrides\": [{ \"name\": \"logging\", \"command\": [\"bundle\", \"exec\", \"rake\", \"publish_weekly_statistics['2018-09-24']\"] }] }" --region eu-west-2
 ```
 
-### Sinatra routes
-
-* `GET /healthcheck` - AWS ELB target group health checking
-* `GET /logging/post-auth/user/?:username?/mac/?:mac?/ap/?:called_station_id?/site/?:site_ip_address?/result/:authentication_result` - Persist a
-  session record with these details
-
 ## Developing
 
 ### Running the tests
 
-You can run the tests and linter with the following commands:
-
 ```shell
 make test
+```
+
+### Using the linter
```

Comment:
Could do a similar thing here as we did for the `make` commands in https://github.com/alphagov/govwifi-authentication-api/pull/60 , think that was a good way to do it.